config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "description": "Description",
+  "plugins": [],
+  "gaCode": ""
+}
```

Comment:
The pull request description says this isn'' required right? Is this an example?

Reply:
Yeah the `config.json` here is just the list of supported keys that can be used